CHANGELOG.md

```diff
@@ -11,6 +11,18 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Added the `updateTrustedDomain` GraphQL API, allowing users to modify a
   trusted domain.
+- Added `IpAddress` GraphQL custom scalar for IP addresses.
+
+  - Applied it to the GraphQL APIs `ipLocation`, `ipLocationList`,
+    `insertAccount`, `updateAccount`, `insertSamplingPolicy`, and
+    `updateSamplingPolicy`.
+  - The API returns the following error message when a value cannot be parsed as
+    an `IpAddr` (e.g., when "abc" is given):
+
+    ```text
+    Failed to parse "IpAddress": Invalid IP address: abc (occurred while
+    parsing "[IpAddress!]")
+    ```
```

Comment:
I think markdownlink requires an empty line before a backtick block, and the editor automatically inserts an empty line 15. However, these empty lines don't look good. Therefore, I suggest:

```
- Added `IpAddress` GraphQL custom scalar for IP addresses
  - Applied it to the GraphQL APIs `ipLocation`, `ipLocationList`,
    `insertAccount`, `updateAccount`, `insertSamplingPolicy`, and
    `updateSamplingPolicy`.
  - The API returns the following error message when a value cannot be parsed as
    an `IpAddr` (e.g., when "abc" is given):
    <!-- markdownlint-disable MD031 -->
    ```text
    Failed to parse "IpAddress": Invalid IP address: abc (occurred while
    parsing "[IpAddress!]")
    ```
```

I believe this could prevent the failure caused by markdownlint.

Reply:
Instead of what I told above, it would better to update the .markdownlint.yaml file as follows:
```yaml
---
MD024:
  siblings_only: true
MD031: false
```

Then, we can change the above part of CHANGELOG like:
```markdown
- Added `IpAddress` GraphQL custom scalar for IP addresses
  - Applied it to the GraphQL APIs `ipLocation`, `ipLocationList`,
    `insertAccount`, `updateAccount`, `insertSamplingPolicy`, and
    `updateSamplingPolicy`.
  - The API returns the following error message when a value cannot be parsed as
    an `IpAddr` (e.g., when "abc" is given):
    ```text
    Failed to parse "IpAddress": Invalid IP address: abc (occurred while
    parsing "[IpAddress!]")
    ```
```

Reply:
It's been changed.
Thanks for your help.